src/playlist-selectors.js

```diff
@@ -315,13 +338,15 @@ export const simpleSelector = function(
 export const lastBandwidthSelector = function() {
   const pixelRatio = this.useDevicePixelRatio ? window.devicePixelRatio || 1 : 1;
 
-  return simpleSelector(
-    this.playlists.master,
-    this.systemBandwidth,
-    parseInt(safeGetComputedStyle(this.tech_.el(), 'width'), 10) * pixelRatio,
-    parseInt(safeGetComputedStyle(this.tech_.el(), 'height'), 10) * pixelRatio,
-    this.limitRenditionByPlayerDimensions
-  );
+  return simpleSelector({
+    master: this.playlists.master,
+    bandwidth: this.systemBandwidth,
+    playerWidth: parseInt(safeGetComputedStyle(this.tech_.el(), 'width'), 10) * pixelRatio,
+    playerHeight: parseInt(safeGetComputedStyle(this.tech_.el(), 'height'), 10) * pixelRatio,
+    playerObjectFit: safeGetComputedStyle(this.tech_.el(), 'objectFit'),
```

Comment:
I think that we might be better to pass an empty string for `playerObjectFit` if `usePlayerObjectFit` is `false` and pass the actual value if `usePlayerObjectFit` is `true`. That keeps `simpleSelector` from having to worry about it at all.

Reply:
This would mean duplicating the logic in `movingAverageBandwidthSelector` and `lastBandwidthSelector`, though, where it not covered by a test - like the pixel ratio logic. Happy to change it if you feel strongly about it.